src/pyfmi/fmi3.pyx

```diff
@@ -1398,7 +1431,20 @@ cdef object _load_fmi3_fmu(
     if model is None:
         _cleanup_on_load_error(fmu_3, context, allow_unzipped_fmu,
                                callbacks, fmu_temp_dir, log_data)
-        raise FMUException("FMU is a {} and not a {}".format(pyfmi_util.decode(FMIL3.fmi3_fmu_kind_to_string(fmu_3_kind)),  pyfmi_util.decode(kind.upper())))
+
+        # TODO, from FMIL we get without blank spaces, i.e. we get 'ModelExchange' and not 'Model Exchange'
+        # when we invoke fmi3_fmu_kind_to_string, perhaps we should format accordingly here?
+        kind_name = kind.upper()
+        if kind.upper() == 'SE':
```

Comment:
I just slightly improved this because otherwise it said "FMU is a ModelEXchange and not a CS" (or SE)